### PR TITLE
Rename iOS classes to MapLibre

### PR DESCRIPTION
**Pull request by kuhnroyal**
Renamed classes and functions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
   packages.
 * Renamed `Maplibre` to `MapLibre` to be in line with maplibre-native 
   (affects for example the classes MaplibreMap and MaplibreMapController).
+* Renamed iOS plugin classes from `Mapbox` to `MapLibre`.
```
Comment (josxha) [inline CHANGELOG.md:25]: I wouldn't consider this change breaking because users won't have to deal with it in any way, or am I wrong?

**Review by josxha — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kuhnroyal) [reply CHANGELOG.md:25]: Actually don't really need to list it in the changelog at all :)

Comment (kuhnroyal) [reply CHANGELOG.md:25]: Moves it to a changes section

Comment (josxha) [reply CHANGELOG.md:25]: That's a point. And if link to the full changelog on GitHub like it is for the last releases, users call still view the complete changelog there.

**Pull request merged**